components/plan-form/PlanForm.js

```diff
@@ -1,12 +1,32 @@
 import Link from "next/link"
-import { useState } from "react"
-import { GlobalContext } from "@/components/GlobalProvider";
-import { useContext } from "react";
-import { addWorkoutDays } from "@/utils/helpers";
+// import { useState } from "react"
+// import { GlobalContext } from "@/components/GlobalProvider";
+// import { useContext } from "react";
+import styled from "styled-components"
+import { useRouter } from "next/router"
+
+const FormContainer = styled.section`
+width: 100vw;
+`
+
+const FormCreatePlan = styled.form`
+width: 100%;
+display: flex;
+flex-direction: column;
+padding: 20px;
+
+& > button {
+    width: fit-content;
+    margin-top: 25px;
+    border: none;
+    padding: 5px 10px;
+    border-radius: 5px;
+}
+`
 
 export default function PlanForm() {
-    const [planData, updatePlanData] = useContext(GlobalContext)
-    
+    // const [planData, updatePlanData] = useContext(GlobalContext)
+    const router = useRouter()
     const handleSave = (event) => {
         event.preventDefault()
```

Comment:
what does this function do? 🤔 

it seems not to serve any purpose yet, so you should remove it